generator/sources/php53/composer.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "kaltura/api-client-library",
```

Comment:
I think the name should contain "5.3", to prevent other and future libraries confusion.